### PR TITLE
feat: Added schema markup class to disable schema using filters

### PR DESCRIPTION
**Pull request by Nikschavan**
Schema can globally be disabled by adding a filter `add_filter( 'astra_schema_enabled', '__return_false' );`

And individual schemas can be disabled by adding a filter `add_filter( 'astra_creativework_schema_enabled', '__return_false' );`

### CODE REVIEW

File: inc/blog/blog-config.php [hunk 1]
```diff
@@ -140,17 +140,32 @@ function astra_post_author( $output_filter = '' ) {
 			echo astra_attr(
 				'post-meta-author',
 				array(
-					'class'     => 'posted-by vcard author',
-					'itemtype'  => 'https://schema.org/Person',
-					'itemscope' => 'itemscope',
-					'itemprop'  => 'author',
+					'class' => 'posted-by author',
```
Comment (premanshup) [inline inc/blog/blog-config.php]: @UttamSharma007 Why are we not including `vcard` CSS?

File: inc/schema/class-astra-creativework-schema.php [hunk 2]
```diff
@@ -0,0 +1,247 @@
...
+ * Astra CreativeWork Schema Markup.
+ *
+ * @since x.x.x
+ */
+class Astra_CreativeWork_Schema extends Astra_Schema {
+
+	/**
+	 * Setup schema
+	 *
+	 * @since 1.0.0
```
Comment (premanshup) [inline inc/schema/class-astra-creativework-schema.php]: @UttamSharma007  This should be `x.x.x`

File: inc/schema/class-astra-creativework-schema.php [hunk 3]
```diff
@@ -0,0 +1,247 @@
...
+	public function article_image_blog_single_post_schema_prop( $attr ) {
+		$attr['itemprop'] = 'image';
+
+		return $attr;
+	}
+
+	/**
+	 * Enabled schema
+	 *
+	 * @since 1.0.0
```
Comment (premanshup) [inline inc/schema/class-astra-creativework-schema.php]: @UttamSharma007 This should be x.x.x

File: inc/schema/class-astra-organization-schema.php [hunk 4]
```diff
@@ -0,0 +1,151 @@
...
+
+		return $attr;
+	}
+
+	/**
+	 * Update Schema markup attribute.
+	 *
+	 * @param  array $attr An array of attributes.
+	 *
+	 * @return string       Updated embed markup.
```
Comment (premanshup) [inline inc/schema/class-astra-organization-schema.php]: `@return` should array. Please check everywhere in the code.

File: inc/blog/blog-config.php (hunk 1 above)
Comment (UttamSharma007) [reply inc/blog/blog-config.php]: Added!

File: inc/schema/class-astra-creativework-schema.php (hunk 2 above)
Comment (UttamSharma007) [reply inc/schema/class-astra-creativework-schema.php]: Done!

File: inc/schema/class-astra-creativework-schema.php (hunk 3 above)
Comment (UttamSharma007) [reply inc/schema/class-astra-creativework-schema.php]: Done!

File: inc/schema/class-astra-organization-schema.php (hunk 4 above)
Comment (UttamSharma007) [reply inc/schema/class-astra-organization-schema.php]: Done!

**Comment by premanshup** [comment]
> @UttamSharma007 It looks like we have not included breadcrumb in this, can you please check? Breadcrumb filter should be also added here.

We have added the breadcrumb schema option to disable.

**Comment by premanshup** [comment]
@UttamSharma007 Please check the Merge Conflict.

**Pull request merged**